src/components/Menu/index.js

```diff
@@ -0,0 +1,219 @@
+            <path d="M67.9684 41.9249C65.8646 42.8502 65.5018 45.9889 67.3517 47.2952C68.9659 48.4563 71.2692 47.8939 72.0672 46.134C72.5206 45.1362 72.5024 44.2835 72.0127 43.3038C71.2873 41.8886 69.4737 41.2718 67.9684 41.9249Z" fill="#313131"/>
+            <path d="M74.3722 49.11C72.2684 50.0353 71.9057 53.174 73.7555 54.4802C75.3697 55.6414 77.673 55.0789 78.471 53.3191C78.9244 52.3212 78.9062 51.4685 78.4166 50.4888C77.6911 49.0737 75.8775 48.4568 74.3722 49.11Z" fill="#313131"/>
+            <path d="M61.3775 49.11C59.2737 50.0353 58.911 53.174 60.7609 54.4802C62.375 55.6414 64.6783 55.0789 65.4763 53.3191C65.9297 52.3212 65.9116 51.4685 65.4219 50.4888C64.6965 49.0737 62.8829 48.4568 61.3775 49.11Z" fill="#313131"/>
+            <path d="M68.0602 56.2877C65.9564 57.213 65.5936 60.3517 67.4435 61.658C69.0577 62.8191 71.361 62.2567 72.1589 60.4968C72.6124 59.499 72.5942 58.6463 72.1045 57.6666C71.3791 56.2514 69.5655 55.6346 68.0602 56.2877Z" fill="#313131"/>
+            <path d="M24.8886 41.7493C24.7401 41.9182 24.6658 42.9104 24.6658 44.8103V47.6179H22.1969C20.5262 47.6179 19.6537 47.7024 19.5052 47.8713C19.1897 48.2301 19.1897 55.0276 19.5052 55.3865C19.6537 55.5554 20.5262 55.6398 22.1969 55.6398H24.6658V58.4686C24.6658 60.6007 24.7215 61.3395 24.9257 61.5295C25.0928 61.6984 26.188 61.7617 28.3971 61.7195L31.6271 61.6562L31.6828 58.6374L31.7385 55.6398H34.3373H36.9176L37.1218 55.0698C37.4188 54.1832 37.2889 48.5679 36.9733 48.0401C36.732 47.6602 36.4907 47.6179 34.2074 47.6179H31.7199V44.8103C31.7199 42.9104 31.6456 41.9182 31.4971 41.7493C31.3486 41.5804 30.3462 41.496 28.1929 41.496C26.0395 41.496 25.0371 41.5804 24.8886 41.7493ZM30.7917 43.0793V43.7126H28.7869H26.8006L26.7449 46.8158L26.6892 49.9401H24.0161H21.3244V52.2622V54.5843H20.7675H20.2106V51.6289V48.6734H22.9023H25.594V45.5491V42.446H28.1929H30.7917V43.0793Z" fill="#313131"/>
+            <path d="M41.669 58.4263C41.2421 59.1018 41.2977 59.5874 41.8361 60.1996C42.3002 60.7273 42.3373 60.7273 44.9176 60.664C47.2565 60.6007 47.5535 60.5584 47.7948 60.1996C48.1475 59.6296 48.129 58.7219 47.7206 58.3208C47.4236 58.0252 46.9224 57.9619 44.6577 57.9619C42.0774 57.9619 41.9289 57.983 41.669 58.4263Z" fill="#313131"/>
+            <path d="M50.5975 58.3419C50.2077 58.8274 50.1891 59.6507 50.5418 60.1996C50.7831 60.5584 51.0801 60.6007 53.4191 60.664C55.9993 60.7273 56.0364 60.7273 56.5005 60.1996C57.0388 59.5874 57.0945 59.1018 56.6676 58.4263C56.4077 57.983 56.2592 57.9619 53.6418 57.9619C51.2101 57.9619 50.8574 58.0041 50.5975 58.3419Z" fill="#313131"/>
+            </svg>
```

Comment:
Adicionar arquivo separado para todos os svg

Reply:
Se for feito isso não é possivel estilizar o path